data/bactrian.txt

```diff
@@ -0,0 +1,419 @@
+			
+			branch destroyed
+				has "Bactrian Unlock [0]: Pirates: done"
+			`	You mention the multiple heavy warships that you saw in the system, and the fact that there are probably many more waiting on Haven.`
+				goto next
+			
+			label destroyed
+			`	You mention the multiple heavy warships that you took out in the system, but that there may be many more waiting on Haven.`
+			
+			label next
+			`	After hearing this news, many of the faces in the group turn grim as they realize that they will need to fight to retrieve the Star Queen. `
+			`	"We will need to get a rescue fleet ready. Guards, ready your ships!" The group responds with a collective "Sir, yes sir!" before running off to their ships.`
+			`	Before leaving for his own ship, the man walks up to you and hands you 100,000 credits.`
+			`	"That's for finding the Star Queen. Now, Captain <last>, would you be willing to help us? If the Star Queen is as heavily defended as you say it is then we're going to need every ship we can get."`
+			choice
+				`	"I'd be glad to."`
+				`	"I would rather let the professionals handle things. Good luck."`
+					decline
+			
+			`	"Thank you so much. The plan is to rendezvous with a number of warships on Memory and then head to Alnitak. My superiors received word from the admiral on Farpoint that there are a number of Navy ships at the ready to avenge the ships they lost, so I expect we'll pick up some help there when we refuel. After the fleet is mustered, we go to Arneb." He smiles briefly, adding, "Just make sure that the <npc> makes it onto the planet. She'll be carrying the landing crew, along with myself."`
+			`	The communications device on his belt squawks, and he quickly responds, giving orders to the fleet that will be joining you on the mission. Before you can return to your ship, he says, "Also, since we'll be going into battle together, I might as well introduce myself. I'm Deep Lieutenant Shane Paris. Good luck fighting; we'll meet you in orbit."`
+				accept
+	
+	npc save accompany
+		government Republic
+		personality heroic escort
+		ship "Mule (Heavy)" "D.S.S. Faraday"
+	
+	on visit
+		dialog `You have landed on Haven, but you left the <npc> behind! Wait for the <npc> to enter the system before landing.`
+
+mission "Bactrian Unlock [1]: Pirates"
+	invisible
+	landing
+	to offer
+		has "Bactrian Unlock [0]: done"
+	to fail
+		or
+			has "Bactrian Unlock [3]: done"
+			has "Bactrian Unlock [3-alt]: offered"
+	npc kill
+		personality nemesis staying
+		system Arneb
+		government Pirate
+		ship "Firebird (Plasma)" "The Court Jester"
+		ship "Firebird (Plasma)" "The Knighted One"
+		ship "Headhunter (Particle)" "Cerberus"
+		ship "Headhunter (Particle)" "Charon"
+
+
+
+mission "Bactrian Unlock [1]: Recruit Escorts"
+	name "Deep Security escorts on <planet>"
+	description "Lt. Paris mentioned that Deep Security was prepping an escort squadron. Meet up with them on <destination>, if you want to."
```

Comment:
No, "prepping" seems informal, like military jargon.
If it were part of dialogue, I'd be perfectly fine with that, but this is an Endless Sky mission description.